maquetado/index.html

```diff
@@ -10,5 +10,11 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech">
+    <img class="tech-logo" src="images/react_icon.svg" alt="React's logo">
+    <h1 class="tech-title">ReactJS</h1>
```

Comment:
```suggestion
    <h3 class="tech-title">ReactJS</h3>
```
Only for keeping a correct syntax 😉